### DISCUSSION TITLE
トラックボールの感度調整について

### DISCUSSION THREAD

**Discussion by StudioKinaco** [Yowkees/keyball#222] [Q&A]
最近keyball61を購入しセルフビルドで組み立てました。
基本的にエラーなく組み上がったのですが、トラックボールの感度調整ができず困っています。
ビルドガイドを参照し特殊キーを使用しスクロール感度を上下させても反応がなく、感度の変更が行われていないようです。

組み立ての段階でトラックボール読み取りICのL字ピンヘッダのU字部分を本体側にハンダしてしまったため読み取りICのほうも固定されなくなってしまい、仕方なくそちらもハンダで対応してしまっています。
そのあたりに問題がある可能性があるのでしょうか？

もしICに問題がある場合、読み取りICとL字ピンヘッダのみ購入することは可能でしょうか。

返信お待ちしております！何卒よろしくお願いします

**Comment by StudioKinaco** [comment]
自己解決しました。特殊キーが一つずれた状態で認識していました。なぜかはわかりませんが。。。

**Comment by Yowkees** [comment]
ご連絡が遅くなり申し訳ありませんでした。特殊キーはビルドするＱＭＫのバージョンによりずれるようです。自己解決いただきありがとうございました。
